builtin/providers/google/resource_compute_instance_template.go

```diff
@@ -134,7 +143,6 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 			"automatic_restart": &schema.Schema{
 				Type:       schema.TypeBool,
 				Optional:   true,
-				Default:    true,
```

Comment:
Nice work! One last little change, since your adding a default to `preemptible`, could you make it clear that the default is `false` in the docs? https://github.com/hashicorp/terraform/blob/master/website/source/docs/providers/google/r/compute_instance_template.html.markdown